scripts/lib/CIME/check_lockedfiles.py

```diff
@@ -122,15 +123,13 @@ def check_lockedfiles(caseroot=None):
                            " recover the original copy from LockedFiles")
                 elif objname == "env_build":
                     logging.warn("Setting build complete to False")
-                    f1obj.set_value("BUILD_COMPLETE", False)
+                    case.set_value("BUILD_COMPLETE", False)
                     if "PIO_VERSION" in diffs.keys():
-                        f1obj.set_value("BUILD_STATUS", 2)
-                        f1obj.write()
+                        case.set_value("BUILD_STATUS", 2)
                         logging.critical("Changing PIO_VERSION requires running "
                                          "case.build --clean-all and rebuilding")
                     else:
-                        f1obj.set_value("BUILD_STATUS", 1)
-                        f1obj.write()
+                        case.set_value("BUILD_STATUS", 1)
```

Comment:
Are the changes in the module even related to multi_driver? If not, please open a new issue and submit a PR with these changes.

Reply:
I don't remember and think that its a total waste of time to try to figure that out.

Reply:
Well I do not understand why you have a double standard where it is okay to fix incidental _code_ issues found when working on a new feature but not okay to fix incidental _comment_ or _documentation_ issues found in the same process.
How about:

1. Fix the documentation/comment issues I pointed out
2. Include information about the incidental fixes in the PR text so that there is some record in the log that this was done.

and I will drop my request for proper separation between new features and bug fixes.